enhancements/compound-bundle-constraints.md

```diff
@@ -238,7 +242,7 @@ properties:
 - type: olm.constraint
   value:
     message: Cannot be required for Baz because...
-    none:
+    not:
```

Comment:
I'm a bit torn about what we should do with this example given it is not a useful one because it is basically equivalent of "accepting all solutions except the `not` one. The outcome is a wide net of anything which can be problematic. It is more helpful in combination of wanting a GVK but not from a particular package.
I wonder if it is a good idea to include a statement to recommend users to use it in combination instead of a standalone constraint.

Reply:
I agree and went ahead and added some more context + a new example that makes this clearer.